lib/run.js

```diff
@@ -82,3 +82,19 @@ function isProfileName (profile) {
   }
   return !/[\\\/]/.test(profile);
 }
+
+/**
+ * Concat binary arguments to arguments array.
+ *
+ * @param {Array} args - arguments
+ * @param {string|Array} binArgs - binary arguments
+ * @return {Array} - arguments
+ */
+function concatBinArgs (args, binArgs) {
+  args = Array.isArray(args) && args || [];
```

Comment:
it would be better to use a ternary here rather than rely on assignment-by-side-effect.